readme.md

```diff
@@ -0,0 +1,38 @@
+# UEFI NVMe-oF Boot Implementation
+## Introduction
+This UEFI Nvme-oF Boot Implementation enables a Boot-from-SAN prototype that produces and consumes [ACPI](https://uefi.org/specifications) XSDT [NBFT](https://uefi.org/specs/ACPI/6.5/05_ACPI_Software_Programming_Model.html?highlight=nbft#description-header-signatures-for-tables-reserved-by-acpi) descriptors that are conformant to the [NVM Express Boot Specification 1.0](https://nvmexpress.org/specification/nvme-boot-specification/) and it’s interface with the [NVMe-oF messaging type](https://uefi.org/specs/UEFI/2.10/10_Protocols_Device_Path_Protocol.html#nvme-over-fabric-nvme-of-namespace-device-path) to [UEFI System Specification 2.10](https://uefi.org/specs/UEFI/2.10/).
+ 
+Further implementation details relative to the UEFI NvmeOfDxe driver can be found in the [POC Readme](https://github.com/timberland-sig/edk2/blob/timberland_1.0_final/NetworkPkg/NvmeOfDxe/readme.md).
+ 
+## Known Open Issues and Errata
+ 
+- [ ] IPv6 still has general config and interop issues; primarily:
+    - [ ] IPv6 DHCP6 Type 3/4 missing
```

Comment:
What do you mean with "DHCP6 Type 3/4"? I am unaware of this terminology.

TBH, I suggest shortening this paragraph to just "IPv6 still has general config and interop issues".

Reply:
>What do you mean with "DHCP6 Type 3/4"? I am unaware of this terminology.

Apologies, this was in reference to [DUID type 3 and 4](https://www.rfc-editor.org/rfc/rfc8415.html#section-11.1) support.  It should probably say that. 

> TBH, I suggest shortening this paragraph to just "IPv6 still has general config and interop issues".

I'm fine with that if @mrabeda doesn't think the detail is needed here.  We can/should probably make these GH issues anyways (they are over in edk2-private still and need porting in here).

Reply:
> this was in reference to DUID type 3 and 4 support

I'm confused. I thought we did support DUID-UUID (and I think I argued elsewhere that we don't need DUID-LL and DUID-V, at least it's low prio). Anyway I think this is the least of our problems with IPv6.

Reply:
The current code supports DUID-EN in the same way that iSCSI and HttpBoot do.  Support for UUID is something tbd.

Reply:
covered by collapse now